src/main/java/org/hmhb/csv/DefaultCsvService.java

```diff
@@ -0,0 +1,245 @@
+package org.hmhb.csv;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import java.util.List;
+import java.util.Set;
+
+import com.codahale.metrics.annotation.Timed;
+import org.hmhb.county.County;
+import org.hmhb.program.Program;
+import org.hmhb.programarea.ProgramArea;
+import org.hmhb.user.HmhbUser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Default implementation of {@link CsvService}.
+ */
+@Service
+public class DefaultCsvService implements CsvService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultCsvService.class);
+
+    /**
+     * An injectable constructor.
+     */
+    public DefaultCsvService() {
+        LOGGER.debug("constructed");
+    }
+
+    private String handleString(
+            @Nullable String data
+    ) {
+        if (data == null) {
+            return "";
+        }
+
+        if (data.contains(",") || data.contains("\"")) {
```

Comment:
If a comma or double quote is in the string, then I must quote the string and escape the quote:

Hello, "this" has a comma and double quote -> "Hello, ""this"" has a comma and double quote"

http://stackoverflow.com/questions/17808511/properly-escape-a-double-quote-in-csv
http://stackoverflow.com/questions/10451842/how-to-escape-comma-and-double-quote-at-same-time-for-csv-file